### PR TITLE
Target of Target of Target

### PR DESCRIPTION
**Pull request by attackpure100**
don't know if this already exists or similar is in progress, but here it is. I use it for myself so thought i would add.

### CODE REVIEW

File: api/config.lua [hunk 1]
```diff
@@ -700,7 +712,7 @@ function pfUI:MigrateConfig()
 
   -- migrating old to new unitframe texts (> 3.0.0)
   if checkversion(3, 0, 0) then
-    local unitframes = { "player", "target", "focus", "group", "grouptarget", "grouppet", "raid", "ttarget", "pet", "ptarget", "fallback" }
+    local unitframes = { "player", "target", "focus", "group", "grouptarget", "grouppet", "raid", "ttarget", "pet", "ptarget", "fallback", "tttarget" }
```
Comment (shagu) [inline api/config.lua]: Migrations are not needed, as there was no tttarget in previous versions. The default you defined will be used anyways.

File: api/config.lua [hunk 2]
```diff
@@ -746,7 +758,7 @@ function pfUI:MigrateConfig()
 
   -- migrating legacy buff/debuff naming (> 3.5.0)
   if checkversion(3, 5, 0) then
-    local unitframes = { "player", "target", "focus", "group", "grouptarget", "grouppet", "raid", "ttarget", "pet", "ptarget", "fallback" }
+    local unitframes = { "player", "target", "focus", "group", "grouptarget", "grouppet", "raid", "ttarget", "pet", "ptarget", "fallback", "tttarget" }
```
Comment (shagu) [inline api/config.lua]: In this specific case, it makes sense, as it will adapt old buff settings to all new unitframes.

Comment (shagu) [inline api/config.lua]: This also seems to be safe to removed, the `if entry.buffs` will never catch, as there won't be `pfUI_config.unitframes[unitframe]` where unitframe is `tttarget`.

File: api/config.lua [hunk 3]
```diff
@@ -757,7 +769,7 @@ function pfUI:MigrateConfig()
 
   -- migrating glow settings (> 3.5.1)
   if checkversion(3, 5, 0) then
-    local common = { "player", "target", "ttarget", "pet", "ptarget"}
+    local common = { "player", "target", "ttarget", "pet", "ptarget", "tttarget"}
```
Comment (shagu) [inline api/config.lua]: That's a useful one, keep it :)

File: api/config.lua [hunk 4]
```diff
@@ -798,7 +810,7 @@ function pfUI:MigrateConfig()
 
   -- migrate buff/debuff position settings (> 3.19)
   if checkversion(3, 19, 0) then
-    local unitframes = { "player", "target", "focus", "group", "grouptarget", "grouppet", "raid", "ttarget", "pet", "ptarget", "fallback" }
+    local unitframes = { "player", "target", "focus", "group", "grouptarget", "grouppet", "raid", "ttarget", "pet", "ptarget", "fallback", "tttarget" }
```
Comment (shagu) [inline api/config.lua]: As in previous versions there was no `TOP` or `BOTTOM` settings for tttarget, there is also nothing to be migrated here. Seems to be safe for removal.

**Comment by attackpure100** [comment]
made changes on (fed05788d2550d12ee8308f353744cb99653223a)

**Comment by shagu** [comment]
Thanks, you PR has been squashed and merget by this commit: https://github.com/shagu/pfUI/commit/050790b7e974c9bf1ccc1748cf78d7198845921d

(PS: I made a minor adjustment and disabled the frame by default `visible = 0`)

**Pull request closed without merging**